### PR TITLE
Add desc for params in AWS-SQS

### PR DESCRIPTION
**Pull request by ShirleyDenkberg**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/AWS-SQS/Integrations/AWS-SQS/README.md [hunk 1]
```diff
@@ -7,38 +7,38 @@ For detailed instructions about setting up authentication, see: [AWS Integration
...
-1. Provide access key id and secret key id.
-2. Provide roleARN and access key id and secret key.
-3. Do not provide any credentials or roleARN than permissions pulled from the service metadata.
+1. Provide Access Key ID and Secret Key ID.
+2. Provide Role ARN and Access Key ID and Secret Key ID.
+3. Do not provide any credentials or Role ARN other than permissions pulled from the service metadata.
 
 
 ## Commands
-You can execute these commands from the Cortex XSOAR CLI, as part of an automation, or in a playbook.
```
Comment (Shellyber) [inline Packs/AWS-SQS/Integrations/AWS-SQS/README.md:41]: Why did you remove the CLI option?

File: Packs/AWS-SQS/Integrations/AWS-SQS/README.md [hunk 2]
```diff
@@ -7,38 +7,38 @@ For detailed instructions about setting up authentication, see: [AWS Integration
 ## Configure AWS - SQS on Cortex XSOAR
 
 1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
-2. Search for AWS - SQS.
-3. Click **Add instance** to create and configure a new integration instance.
+3. Search for AWS - SQS.
```
Comment (Shellyber) [inline Packs/AWS-SQS/Integrations/AWS-SQS/README.md:10]: Where did 2 go?

File: Packs/AWS-SQS/Integrations/AWS-SQS/README.md [hunk 3]
```diff
@@ -7,38 +7,38 @@ For detailed instructions about setting up authentication, see: [AWS Integration
...
+    | Timeout | The time in seconds till a timeout exception is reached. You can specify just the read timeout (for example 60) or also the connect timeout followed after a comma (for example 60,10). If a connect timeout is not specified a default of 10 second will be used. | False        |
+    | Retries | The maximum number of retry attempts when connection or throttling errors are encountered. Set to 0 to disable retries. The default value is 5 and the limit is 10. Note: Increasing the number of retries will increase the execution time." More details about the retries strategy is available [here](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/retries.html). | False        |
     | Fetch incidents |  | False        |
-    | max_fetch | Maximum incidents for one fetch | 10           |
-    | First fetch timestamp (&lt;number&gt; &lt;time unit&gt;, e.g., 12 hours, 7 days) |  | False        |
+    | Maximum incidents for one fetch. Hard cap of 100. | Maximum number of incidents for a single fetch. | 10           |
+    | First fetch timestamp | First fetch query <number> <time unit>, e.g., 7 days. Default 3 days  | False        |
```
Comment (Shellyber) [inline Packs/AWS-SQS/Integrations/AWS-SQS/README.md:26]:
```suggestion
    | First fetch timestamp | First fetch query `<number> <time unit>`, e.g., `7 days`. Default `3 days`  | False        |
```

**Review by Shellyber — APPROVED** [review]

**Comment by Shellyber** [comment]
@ShirleyDenkberg Looks good, please see my comments before merging :)

**Comment by ShirleyDenkberg** [comment]
@Shellyber Thanks.  Implemented your changes.
